trunk/source/Library-CHC/META-INF/MANIFEST.MF

```diff
@@ -5,16 +5,18 @@ Bundle-SymbolicName: de.uni_freiburg.informatik.ultimate.lib.chc
 Bundle-Vendor: Ultimate Program Analysis Team
 Bundle-Version: 0.3.0
 Bundle-RequiredExecutionEnvironment: JavaSE-21
-Bundle-ClassPath: .,
- Eldarica-assembly-2.0.8.jar
+Bundle-ClassPath: .
```

Comment:
Is this still necessary if Elderica is no longer included as a local `*.jar`.

Reply:
Yes. it seems like it can be omitted